precompile/modules/initia_stdlib/sources/table.move

```diff
@@ -224,6 +224,50 @@ module initia_std::table {
         (key, &mut box.val)
     }
 
+    public inline fun get_last_key_and_value<K: copy + drop, V>(table: &Table<K,V>): (K,&V) {
```

Comment:
these 3 functions sounds too redundant for me
how about rename it likt get_last(), loop_mut() and loop()?

Reply:
how about `get_last_key_value`?
loop is the duplicated keyword of move lang. so i change loop to walk' `walk` , `walk_mut`

Reply:
the function's prototype - just a single argument and a KV pair as a return - describes what the callers get so get_last_key_value() is still garrulous to me.
how do you think @ALPAC-4? I think you would have an opinion because you did so much about contracts :)

Reply:
Honestly, I'm not sure if we need a function called `get_last`. This seems similar to having borrow last in a vector.